src/icon.rs

```diff
@@ -370,10 +370,10 @@ mod test {
         File::create(&file_path).expect("failed to create file");
         let meta = Meta::from_path(&file_path, false).unwrap();
 
-        let icon = Icons::new(Theme::Fancy);
-        let icon = icon.get(&meta.name);
+        let icon = Icons::new(Theme::Fancy, " ".to_string());
+        let icon_str = icon.get(&meta.name);
 
-        assert_eq!(icon, format!("{}{}", "\u{f016}", ICON_SPACE)); // 
+        assert_eq!(icon_str, format!("{}{}", "\u{f016}", icon.icon_separator)); // 
```

Comment:
Can we also add a test for a non default icon separator is being used.

Reply:
Should I put the test there (`src/icon.rs`) or in the `src/flags/icon.rs` file, where I already created a test for the default case?

Reply:
I think we will keep it in `src/flags/icon.rs`.

Reply:
I made the changes.  You can review again now.